Tox error and styling error in connection with #1722

Styling errors:
```
pydocstyle --convention=numpy scholia
'scholia/utils.py:95 in public function `remove_special_characters_url`:
        D400: First line should end with a period (not 's')
scholia/utils.py:95 in public function `remove_special_characters_url`:
        D205: 1 blank line required between summary line and description (found 0)
```
Tox errors on py27:
```
=============================================================================================================== ERRORS ================================================================================================================
________________________________________________________________________________________________ ERROR collecting scholia/__main__.py _________________________________________________________________________________________________
scholia/__main__.py:31: in <module>
    from .utils import string_to_type
scholia/utils.py:3: in <module>
    import urllib.parse
E   ImportError: No module named parse
___________________________________________________________________________________________________ ERROR collecting scholia/qs.py ____________________________________________________________________________________________________
scholia/qs.py:6: in <module>
    from .utils import escape_string
scholia/utils.py:3: in <module>
    import urllib.parse
E   ImportError: No module named parse
__________________________________________________________________________________________________ ERROR collecting scholia/utils.py __________________________________________________________________________________________________
scholia/utils.py:3: in <module>
    import urllib.parse
E   ImportError: No module named parse
________________________________________________________________________________________________ ERROR collecting scholia/app/views.py ________________________________________________________________________________________________
scholia/app/views.py:24: in <module>
    from ..utils import sanitize_q, remove_special_characters_url
scholia/utils.py:3: in <module>
    import urllib.parse
E   ImportError: No module named parse
_______________________________________________________________________________________________ ERROR collecting scholia/scrape/nips.py _______________________________________________________________________________________________
scholia/scrape/nips.py:49: in <module>
    from ..qs import paper_to_quickstatements
scholia/qs.py:6: in <module>
    from .utils import escape_string
scholia/utils.py:3: in <module>
    import urllib.parse
E   ImportError: No module named parse
_______________________________________________________________________________________________ ERROR collecting scholia/scrape/ojs.py ________________________________________________________________________________________________
scholia/scrape/ojs.py:34: in <module>
    from ..qs import paper_to_quickstatements
scholia/qs.py:6: in <module>
    from .utils import escape_string
scholia/utils.py:3: in <module>
    import urllib.parse
E   ImportError: No module named parse
!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!! Interrupted: 6 errors during collection !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
======================================================================================================= 6 error in 0.45 seconds =======================================================================================================
ERROR: InvocationError for command '/home/faan/projects/scholia/.tox/py27/bin/python -m pytest --doctest-modules scholia' (exited with code 2)
```

One quick fix would be to just remove testing on py27. The workflows is testing on 3.7, 3.8, 3.9 Python versions.

--- a/scholia/utils.py
+++ b/scholia/utils.py
@@ -92,16 +92,18 @@ def string_to_type(string):
 
 
 def remove_special_characters_url(url):
-    """Remove url encoded characters and normalize non-ascii characters
+    """Remove url encoded characters and normalize non-ascii characters.
+
     Parameters
     ----------
     url : str
-        url encoded string
+        URL-encoded string
 
     Returns
     -------
     formatted_string : str
         Normalized string without non-ascii characters or spaces
+
     """
     decoded_url = urllib.parse.unquote(url)
     encode_string = decoded_url.encode("ascii", "ignore")

--- a/tox.ini
+++ b/tox.ini
@@ -1,5 +1,5 @@
 [tox]
-envlist = flake8, pydocstyle, py27, py35, py37
+envlist = flake8, pydocstyle, py35, py37
 
 
 [testenv:py27]

